SETUP_SUBMITTER_CMF.md

```diff
@@ -54,9 +55,43 @@ runuat.bat BuildPlugin -plugin="C:\deadline\deadline-cloud-for-unreal-engine\src
 
 - Copy the “package” folder above to your Unreal installation’s Plugins folder (C:\Program Files\Epic Games\UE_5.4\Engine\Plugins\UnrealDeadlineCloudService for example)
 
-## Submitter Installer
+## Python Dependencies
+
+There are several ways to install the required Python dependencies.
+
+If you've built and installed the plugin from the release branch above, you can simply install from pip.  Either install as: (Adjust paths accordingly)
+
+```
+"C:\Program Files\Epic Games\UE_5.4\Engine\Binaries\ThirdParty\Python3\Win64\python" -m pip install deadline-cloud-for-unreal-engine --target "C:\Program Files\Epic Games\UE_5.4\Engine\Plugins\UnrealDeadlineCloudService\Content\Python\libraries"
+``
+
+Or alternatively in your .uplugin file (In the above steps this would live at C:\Program Files\Epic Games\UE_5.4\Engine\Plugins\UnrealDeadlineCloudService\UnrealDeadlineCloudService.uplugin) you can add a "PythonRequirements" section which matches the latest release of deadline-cloud-for-unreal-engine in GitHub/PyPi, for example:
+
+```
+	"PythonRequirements":
+	[
+		{
+
+			"Platform": "All",
+			"Requirements":
+			[
+				"deadline-cloud-for-unreal-engine>=0.3.0"
+			]
+		}
+	]
+```
+
+Note that you may wish to disable the "strict hash" feature in Unreal's Python settings, or add hash settings for specific library and dependency versions you wish to consume.
+
+Finally if you're pulling from mainline you may have python dependencies which are not yet released to PyPi - you'll want to run something like this to build and install your mainline version.  Note that the .whl file will need to be changed to reflect the version which is output by hatch build:
+
+```
+pip install hatch (If not installed)
```

Comment:
nit: can we move `(If not installed)` into a comment?

Reply:
Sure!